From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Tue, 25 Dec 2012 00:27:23 +0100
Subject: bin symlink

Simplify installed symlink from bin/s9e-core to s9e by removing
absolute prefix, which is wrong when DESTDIR is turned on, and
problematic if the target bin directory is moved.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 520041b..e51eb06 100644
--- a/Makefile
+++ b/Makefile
@@ -167,7 +167,7 @@ install-util:
 install-s9e:	s9e-core.image
 	install -d -m 0755 $(BINDIR)
 	install -d -m 0755 $(LIBDIR)
-	ln -fs $(BINDIR)/s9 $(BINDIR)/s9e-core
+	ln -fs s9 $(BINDIR)/s9e-core
 	install $C -m 0644 s9e-core.image $(LIBDIR)
 	sed -e "s|^#! /usr/local|#! $(PREFIX)|"	\
 		<prog/s9e1.scm >$(BINDIR)/s9e
